src/room/GroupCallView.tsx

```diff
@@ -165,6 +167,22 @@ export const GroupCallView: FC<Props> = ({
   const latestDevices = useLatest(deviceContext);
   const latestMuteStates = useLatest(muteStates);
 
+  const enterRTCSessionOrError = async (
+    rtcSession: MatrixRTCSession,
+    perParticipantE2EE: boolean,
+  ): Promise<void> => {
+    try {
+      await enterRTCSession(rtcSession, perParticipantE2EE);
+    } catch (e) {
+      if (e instanceof ElementCallError) {
+        // e.code === ErrorCode.MISSING_LIVE_KIT_SERVICE_URL)
+        setEnterRTCError(e);
+      } else {
+        logger.error(`Unknown Error while entering RTC session`, e);
```

Comment:
Should we be re-throwing the error here which I *think* would make it be the same as the *current* error behaviour?

```suggestion
        logger.error(`Unknown Error while entering RTC session`, e);
        // TODO: rather than re-throw we probably want to display some generic
        // error message
        throw e;
```

Reply:
No it depends on when this call is made. 
If you re-throw:
- and it is called during rendering it will crash the component thus triggering the Error Boundary
- and it is called on a button click, it will not crash rendering, so error boundary will not ne triggered

Reply:
So, are you saying that the way it is written the behaviour is not changed in the case of a non-`ElementCallError` being encountered?

Reply:
Depending on where it was called yes. But maybe I could just turn it into a new Unknown ElementCallError?
let me try something. Will reply with the commit

Reply:
@hughns 
 https://github.com/element-hq/element-call/pull/3027/commits/1fbf8e268622733b6569430859d16d924fee4759
 
 With that we will capture any kind of errors, it will be properly reported to Sentry, though the user facing message will not be very usefull

Reply:
Okay, this feels better than before. We can consider it further as we do more work in later PRs 👍